src/Admin.php

```diff
@@ -12,6 +12,20 @@
  */
 class Admin {
 
+  /**
+   * Cron event name for revision cleanup.
+   *
+   * @var string
+   */
+  const CRON_EVENT_REVISION_CLEANUP = Plugin::PREFIX . '/cron/revision-cleanup';
+
+  /**
+   * Time-frame in days for which to retain all revisions (for post type 'posts).
+   *
+   * @var int
+   */
+  const CRON_EVENT_REVISION_CLEANUP_RETAIN_DAYS = 90;
```

Comment:
IMO, this should be a filterable value as some instances might not have a another revision frequency and want to store revisions much longer/shorter.